test_scripts/API/VehicleData/Versioning/Version_5_0_and_higher/005_SubscribeVehicleData_OneParam_Version_5_0.lua

```diff
@@ -0,0 +1,72 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2997
+--
+-- Description:
+--   SDL sends SubscribeVehicleData request with fuelRange param to HMI
+--
+-- Expected behavior:
+--   SDL sends request to HMI with all params, requested from mobile
+--
+-- Steps to reproduce :
+--   1. Mobile application sends valid GetVehicleData request to SDL with params:
+--      1) fuelRange, that is available from syncMsgVersion 5.0 only;
+--
+--   2.SDL must:
+--     1) Transfer GetVehicleData request to HMI 
+--     2) After successful response from HMI response SUCCESS to Mobile App, 
+--        with parameter value received from HMI
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/Versioning/common')
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+--[[ Local Variables ]]
+local rpc = {
+  name = "SubscribeVehicleData",
+  params = {
+    fuelRange = true -- available since 5.0
+  }
+}
+
+local vehicleDataResults = {
+  fuelRange = {
+    dataType = "VEHICLEDATA_FUELRANGE", 
+    resultCode = "SUCCESS"
+  }
+}
+
+--[[ Local Functions ]]
+local function processRPCFailure()
```

Comment:
@YarikMamykin This function is the same as `processRPCSuccess` from previous scripts and `resultCode = "SUCCESS"` . Why is it called `processRPCFailure`?